xapi-profiles-structure.md

```diff
@@ -452,12 +463,17 @@ Property | Type | Description | Required
 
 A Statement Template MUST NOT have both `objectStatementRefTemplate` and `objectActivityType`.
 
-The verb, object activity type, attachment usage types, and context activity types listed are called Determining Properties.
+The verb, object activity type, homepage, extensions, category context activity, attachment usage types, and context activity types listed are called Determining Properties.
 
-A Profile Author MUST change a Statement Template's `id` between versions if any of the Determining Properties, StatementRef properties, or rules change. Changes of `scopeNote` are not considered changes in rules.
+A Profile Author MUST change a Statement Template's `id` between versions if any of the Determining Properties, StatementRef properties, or rules change. 
+Changes of `scopeNote` are not considered changes in rules.
+A Profile Author SHOULD create a Statement Template with a single Determining Property of the Category Context Activity
```

Comment:
I think this is limiting / should be reworded to something like,

"A Profile Author SHOULD create Statement Templates with the profileReference Determining Property containing at least the Statement Templates' inScheme"

Reply:
Per 6/7/2022, Change the notion of lines 470-471 to say that the Profile should include at least a Statement Template that does those things.  Capture that there are valid profiles without Statement Templates (however, this must be justified to the inherent inclusion of the context category context activity id in each Statement Template), Also do like the addition proposed by @FeLungs